### PR TITLE
Display authentication type valid values

### PR DESCRIPTION
**Pull request by mooreds**
Previously we were only showing the authenticationType valid values on the token claims page, but it is used in other places:

* login webhooks
* login validation lambda response

This PR does a few things:

* reworks the token claims page to use the AvailableSince component for more consistent styling
* add the authenticationType values to the missing areas mentioned above
* changes the authenticationType include file to be better named (no more underscores!)
* modifies the authenticationType include to take a show_since value, because as types and features are added, displaying availableSince doesn't make sense if the authentiationType was added before the feature was.

### CODE REVIEW

**Comment by mooreds** [comment]
eslint fails because it is trying to operate on a file I removed. still trying to figure out how to get gh cli to exclude such files.

**Review by lyleschemmerling — APPROVED** [review]
I fixed the eslint issues and used javascript since this is a js repo

**Comment by mooreds** [comment]
I thought this was a markdown repo @lyleschemmerling  :)

File: astro/src/content/docs/_shared/_authentication-type-values.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* `PING` - The user was authenticated using a `PUT` request on the Login API. This is used to record a login event without prompting for credentials, such as when the FusionAuth SSO session is used.
+* `REFRESH_TOKEN` - The User requested a new JWT using a Refresh Token.
+* `REGISTRATION` - The user was created using the Registration API. {props.show_since < 101600 ? <AvailableSince since="1.16.0" /> : ''}
+* `SAMLv2` - The User was authenticated using an external SAMLv2 provider. {props.show_since < 100600 ? <AvailableSince since="1.6.0" /> : ''}
+* `SAMLv2IdpInitiated` - The User was authenticated using an external SAMLv2 provider using an IdP Initiated login. {props.show_since < 102800 ? <AvailableSince since="1.28.0" /> : ''}
+* `SonyPSN` - The User was authenticated using Sony {props.show_since < 102800 ? <AvailableSince since="1.28.0" /> : ''}
+* `Steam` - The User was authenticated using Steam {props.show_since < 102800 ? <AvailableSince since="1.28.0" /> : ''}
+* `TWITTER` - The User was authenticated using Twitter. {props.show_since < 100100 ? <AvailableSince since="1.1.0" /> : ''}
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/docs/_shared/_authentication-type-values.mdx:33]: Yeah. I filed an issue for updating things: https://github.com/FusionAuth/fusionauth-issues/issues/2942

[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/docs/_shared/_authentication-type-values.mdx:33]: We'd probably leave this as-is for backwards compatibility anyway.

**Review by hjaret — APPROVED** [review]

**Review by lyleschemmerling — APPROVED** [review]

**Pull request merged**